packages/api/amplify_api/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.0-next.5+1
+
+- Minor bug fixes and improvements
```

Comment:
request: Could this include the fix in changelog of amplify_api_dart?

Reply:
Updated 👍 will look into automating this